styleguide/derek/examples/_homepage.html

```diff
@@ -223,40 +231,48 @@ <h2 class="right">FANATICAL SUPPORT WhereVER YOU ARE, DaY OR NIGHT, WE HAVE YOU
   </div>
   
   <!-- logo river -->
-  <div class="container-fluid half-padding bg-light-gray">
-    <div class="row">
-      <div class="col-xs-12">
-        <ul class="logos">
-          <li><img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/underarmor.png" alt=""></li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/yeti-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/rip-curl-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/11/six-flags-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/xero-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/kendra-scott-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/gsdm-logo.png</li>
-          <li></li>
-        </ul>
+  <div class="container-fluid half-padding bg-blue-gray">
+    <div class="container customer_stories_logos">
+      <div class="row">
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/underarmor.png" alt="">
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/yeti-logo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/rip-curl-logo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/six-flags-logo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/ksLogo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/razorfish-logo.png"/>
+        </div>
```

Comment:
@ejchiefs Can we have a ticket in the backlog for adding alt to all image tags for accessibility 👍